src/api.js

```diff
@@ -484,6 +484,11 @@
       bus.subscribe(this.getEventName(connect.ContactEvents.CONNECTED), f);
    };
 
+   Contact.prototype.onSession = function(f) {
+      var bus = connect.core.getEventBus();
+      bus.subscribe(this.getEventName(connect.ContactEvents.SESSION), f);
```

Comment:
Contacts are very generic and not supposed to handle the media specific operations (here it takes the RTC session / voice call media specific)  - i would suggest this to be put under connections API which is handling media specific operations like hold resume on voice call.